app/views/users/show.html.erb

```diff
@@ -1 +1,18 @@
-<h1>User Details</h1>
\ No newline at end of file
+<h1>Single user</h1>
+
+    <img src="<%= @user.photo %>" alt="user photo">
+    <h2> <%= @user.name %></h2>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+ 
+  <h2>Bio</h2>
+  <p><%= @user.bio %></p>
+  <% @user.most_recent_posts.each do |post| %>
+      <h2>Post #<%= post.id %></h2>
+      <p><%= post.text %></p>
+        <span>Comments: <%= "#{post.comments_counter || 0 } "%></span>
+        <span>, Likes: <%= "#{ post.likes_counter || 0 }"%></span>
+  <% end %> 
+    <a href=<%= "/users/#{@user.id}/posts"%>>
+    See All Posts
+  </a>

```

Comment:
- [ ] This button should fetch all the posts for a user, unfortunately, I got only the first three posts. I have attached an image for your reference.

![posts](https://user-images.githubusercontent.com/53124548/230358850-405a44f2-3790-43f6-97c5-2bd742728a1a.png)